tests/integration/test_sts.py

```diff
@@ -173,3 +180,15 @@ def test_custom_caller_identity(self, sts_client, iam_client, monkeypatch):
         response = iam_client.get_user()
         assert response["User"]["UserName"] == test_name
         assert response["User"]["UserId"] == test_id
+
+    def test_expiration_date_format(self):
```

Comment:
Any reason this isn't written in a "parity-style"? (I.e. as a test that could be executed against AWS as well for validation)

Reply:
The main reason is that the request with `boto3` would raise the exception contained in the PR description, so we'd have to use some more boto hooks to access the response content, to run the assertions. Tagged the test as `@only_localstack`, though. 👍